lib/rules/length-zero-no-unit/README.md

```diff
@@ -1,4 +1,4 @@
-# length-zero-no-unit
+  # length-zero-no-unit
```

Comment:
```suggestion
# length-zero-no-unit
```

Reply:
Another reason to work in stylelint repo, and not in a fork:

<img width="787" alt="Screenshot 2019-08-11 at 12 15 52" src="https://user-images.githubusercontent.com/654597/62832637-cd94a100-bc31-11e9-9c5f-73da00955b58.png">

I could make this change myself and merge PR. But now we need to wait for you to accept suggestion, and only then we can merge :)

Reply:
fixed

Reply:
> I could make this change myself and merge PR. But now we need to wait for you to accept suggestion, and only then we can merge :)

for sure the most valued advantage =)